### DISCUSSION TITLE
Italian localization

### DISCUSSION THREAD

**Discussion by MassiminoilTrace** [mrgnr/camus#12] [Ideas]
Hi!
I really like this project, it's the only video server that can run on my raspberry thanks to its p2p nature.

I would like to help in some way.
I'm not sure to be proficient enough to contribute directly to the code. I used Flask in the past, but it would take me time to dive into your code.
By the way, when the project will have some localization support, I can translate it to Italian for sure!

**Comment by mrgnr** [comment]
Hi @MassiminoilTrace, glad you find the project useful!

Localization would be a great feature to have. I'd be willing to take contributions for localization support, as I don't have much experience in that area. Translators would be great to have too!

In terms of implementation, there is both server-generated content and content in the React app that would need to be localized. Would probably use [Flask-Babel](https://flask-babel.tkte.ch/) for server content and [react-i18next](https://github.com/i18next/react-i18next) for React. Fortunately Camus doesn't contain that much text needing to be translated, so I think this should be a straightforward feature to add.
